src/components/builds/ConfigurationWithIssues.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { createStyles, Theme } from '@material-ui/core';
+import { WithStyles, withStyles } from '@material-ui/core/styles';
+import { Alert } from '@material-ui/lab';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    configurationTable: {
+      color: theme.palette.type === 'dark' ? theme.palette.primary.light : theme.palette.primary.dark,
+      background: theme.palette.type === 'dark' ? theme.palette.primary.dark : theme.palette.primary.light,
+      width: '100%',
+      borderSpacing: '0px',
```

Comment:
can be probably just `0`

```suggestion
      borderSpacing: '0',
```

Reply:
Changed in https://github.com/cirruslabs/cirrus-ci-web/pull/351/commits/3609ad2e4486faa860fb33012299971e441c63b9.